src/Workspaces/Core/Portable/Extensions/OperationExtensions.cs

```diff
@@ -60,7 +61,11 @@ public static ValueUsageInfo GetValueUsageInfo(this IOperation operation)
                         //
                         return ValueUsageInfo.Write;
 
+#if CODE_STYLE
+                    case IOperation iop when iop.GetType().GetInterfaces().Any(i => i.Name == "IRecursivePatternOperation"):
+#else
```

Comment:
IMO we should not be porting until we're on a version that supports what the feature needs.  reflection here is just busted.